packages/aragon-wrapper/src/index.js

```diff
@@ -454,6 +458,122 @@ export default class Aragon {
+      name: 'local',
+      provider: new LocalIdentityProvider()
+    }]
+    // TODO: detect other installed providers
+    const detectedIdentityProviders = []
+    const identityProviders = [...defaultIdentityProviders, ...detectedIdentityProviders]
+
+    // Init all providers
+    await Promise.all(identityProviders.map(({ provider }) => {
+      // Most providers should have this defined to a noop function by default, but juts in case
+      if (typeof provider.init === 'function') {
+        return provider.init()
+      }
+      return Promise.resolve()
+    }))
+
+    this.identityProviderRegistrar = new Map(
+      identityProviders.map(({ name, provider }) => [name, provider])
+    )
+    // Set up identity modification intent observable
+    this.identityIntents = new Subject()
+  }
+
+  /**
+   * Modify the identity metadata for an address using the requested provider.
+   *
+   * @param  {string} address Address to modify
+   * @param  {Object} metadata Modification metadata object
+   * @param  {string} [providerName='local'] Name of the identity provider to use
+   * @return {Promise<void>}
+   */
+  modifyAddressIdentity (address, metadata) {
+    const providerName = 'local' // TODO - get provider
+    const provider = this.identityProviderRegistrar.get(providerName)
+    if (provider && typeof provider.modify === 'function') {
+      return provider.modify(address, metadata)
+    }
+    throw new Error(`Provider (${providerName}) not installed`)
+  }
+
+  /**
+   * Resolve the identity metadata for an address using the requested provider.
+   *
+   * @param  {string} address Address to resolve
+   * @return {void}
+   */
+  resolveAddressIdentity (address) {
+    const providerName = 'local' // TODO - get provider
+    const provider = this.identityProviderRegistrar.get(providerName)
+    if (provider && typeof provider.resolve === 'function') {
+      return provider.resolve(address)
+    }
+    throw new Error(`Provider (${providerName}) not installed`)
+  }
+
+  /**
+   * Request an identity modification using the requested provider.
+   *
+   * @param  {string} address Address to modify
+   * @return {void}
+   */
+  requestAddressIdentityModification (address) {
+    const providerName = 'local' // TODO - get provider
+    if (this.identityProviderRegistrar.has(providerName)) {
+      return new Promise((resolve, reject) => {
+        this.identityIntents.next({
+          address,
+          providerName,
+          resolve,
+          reject
+        })
+      })
+    }
+
+    throw new Error(`Provider (${providerName}) not installed`)
+  }
+
+  /**
+   * Request an identity modification using the requested provider.
+   *
+   * @param  {string} [providerName='local'] Name of the identity provider to use
+   * @return {Promise<void>}
+   */
+  clearLocalIdentities () {
+    return this.identityProviderRegistrar.get('local').clear()
+  }
+
+  /**
+   * Request an identity modification using the requested provider.
```

Comment:
Let's make sure to update the docstrings of `getLocalIdentities()` and `clearLocalIdentities()`